doc/source/whatsnew/v0.21.0.txt

```diff
@@ -76,6 +76,7 @@ Removal of prior version deprecations/changes
 
 - ``pd.read_excel()`` has dropped the ``has_index_names`` parameter (:issue:`10967`)
 - ``Categorical`` has dropped the ``.order()`` and ``.sort()`` methods in favor of ``.sort_values()`` (:issue:`12882`)
+- ``pd.eval`` and ``DataFrame.eval`` have changed the default of ``inplace`` from ``None`` to ``False`` (:issue:`11149`)
```

Comment:
Hmm...that's new (you could also make the argument to that for `pd.read_csv` for example).  What's the correct format:

``pd.eval`` --> :func:`eval`
``DataFrame.eval`` --> ???